portal/v2/src/ClusterDetailList.tsx

```diff
@@ -70,6 +71,12 @@ export class ClusterDetailComponent extends Component<ClusterDetailComponentProp
             <MachineSetsWrapper currentCluster={this.props.cluster!} detailPanelSelected={this.props.detailPanelVisible} loaded={this.props.isDataLoaded}/>
           );
         }
+        case "clusteroperators":
+        {
+          return (
+            <ClusterOperatorsWrapper currentCluster={this.props.cluster!} detailPanelSelected={this.props.detailPanelVisible} loaded={this.props.isDataLoaded}/>
```

Comment:
Can we truly expect cluster to be defined? if so I think it would better to make a change to the type or add a conditional to do type narrowing (https://www.typescriptlang.org/docs/handbook/2/narrowing.html).

Reply:
Have added a check for cluster defined prior to this component